src/serverMain/kotlin/com/studo/campusqr/endpoints/AccessManagement.kt

```diff
@@ -46,7 +44,7 @@ suspend fun AuthenticatedApplicationCall.listAccess() {
   val accessPayloads: List<BackendAccess> = runOnDb {
     with(getCollection<BackendAccess>()) {
       when {
-        locationId != null && (user.isModerator) -> {
+        locationId != null && (user.canEditLocations) -> {
```

Comment:
why do we need here editLocation permission?

Reply:
because with EDIT_ACCESS (ACCESS_MANAGER) you can only list your own BackendAccesses and not the BackendAccesses created by other people